BridgeAppSDK/ORKResult+SBAExtension.swift

```diff
@@ -280,12 +280,10 @@ class AnswerKeyAndValue: NSObject {
     }
 }
 
-protocol ORKQuestionResultAnswerJSON {
+public protocol ORKQuestionResultAnswerJSON {
     func jsonSerializedAnswer() -> AnswerKeyAndValue?
 }
 
-private var _timeOnlyFormatter: NSDateFormatter? = nil
-
 extension ORKQuestionResult: ORKQuestionResultAnswerJSON {
```

Comment:
it won't let me declare extensions that implement protocols as public. I assume they match the public-ness of the protocol.